saleor/graphql/product/tests/test_product.py

```diff
@@ -4061,6 +4061,40 @@ def test_query_product_image_by_invalid_id(
     assert content["data"]["product"]["imageById"] is None
 
 
+@pytest.mark.parametrize(
+    "sort_by",
+    ({"field": "ID", "direction": "ASC"}, {"field": "ID", "direction": "DESC"}),
```

Comment:
You could also add a test that if you don't provide any sort input, then the default sorter is used (media has field order)